easybuild/easyconfigs/h/Harminv/Harminv-1.4.2-foss-2023b.eb

```
@@ -0,0 +1,30 @@
+easyblock = 'ConfigureMake'
+
+name = 'Harminv'
+version = '1.4.2'
+
+homepage = 'https://github.com/stevengj/harminv'
+description = """Harminv is a free program (and accompanying library) to solve the problem of harmonic inversion -
+ given a discrete-time, finite-length signal that consists of a sum of finitely-many sinusoids (possibly exponentially
+ decaying) in a given bandwidth, it determines the frequencies, decay constants, amplitudes, and phases of those
+ sinusoids."""
+
+toolchain = {'name': 'foss', 'version': '2023b'}
+toolchainopts = {'opt': True, 'unroll': True, 'pic': True, 'cstd': 'c99'}
+
+source_urls = ['https://github.com/stevengj/harminv/releases/download/v%(version)s/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['5a9a1bf710972442f065d0d62c62d0c4ec3da4a3696d7160a35602c9470bc7a2']
+
+builddependencies = [
+    ('pkgconfig', '1.5.5', '-python'),
+]
+
+configopts = "--with-pic --with-blas=${EBROOTOPENBLAS} --with-lapack=${EBROOTSCALAPACK} --enable-shared"
```

shouldn't you use flexiblas here as well?
```suggestion
configopts = "--with-pic --with-blas=flexiblas --with-lapack=flexiblas --enable-shared"
```

---

Oh, I'm sorry, this shoud not be in here at all. I wanted to only hand in the gfbf variant :| sorry about that.